Cargo.toml

```diff
@@ -1,23 +1,29 @@
 [package]
 name = "multihash"
-description = "Implementation of the multihash format"
-repository = "https://github.com/multiformats/rust-multihash"
-keywords = ["multihash", "ipfs"]
 version = "0.9.4"
 authors = ["dignifiedquire <dignifiedquire@gmail.com>"]
+edition = "2018"
```

Comment:
This is not a big deal. I just wonder why did you change the ordering of the keys. Such cleanups should be a separate commit as they are not related to the actual change.

Reply:
This is just what I saved just now to resolve the conflict :rofl: 
I will revert it.